src/containers/Header/index.js

```diff
@@ -16,11 +16,46 @@ export function Header(props) {
 	const href = props.brand_image.href ?? "/";
 
 	const [headerProperties, setHeaderProperties] = useState(false);
+	const { width } = useWindowDimensions();
+
+	function useWindowDimensions() {
+		const [windowDimensions, setWindowDimensions] = useState(
+			getWindowDimensions()
+		);
+
+		useEffect(() => {
```

Comment:
@Aringoaway You should not use hooks in nested functions: https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

Instead you can use `useCallback` component https://reactjs.org/docs/hooks-reference.html#usecallback